### PR TITLE
Completed Task 776 : Design System > Elements > Enhancements in Toast

### LINKED ISSUE #776: Design System > Elements > Enhancements in Toast
Design System > Elements > Enhancements in Toast
Docs	1]  Toast development different than design
Default	2] 4 types of Layouts
	3] Please compare the colors which is required for element
https://www.figma.com/design/vziFLZAgMFi8wA5SlikLh5/Raaghu-Design-System?node-id=1601-27117&node-type=frame&t=ncBHc3XyPjDjE2We-0

### LINKED ISSUE #823: Design System > Create color variants for toast elements
Design System > Create color variants for toast elements

### PR DESCRIPTION
**Pull request by DeepashriDatar**
## Description

> Added new properties to match the wireframe 
> Added 4 stories Default, Download, Chat & Request

## ScreenShot 
1. Default (With Header):
image

2.  Default (Without Header):
image

3. Toast with Download (With Header)
image
  
4. Toast with Download (Without Header)
image

5. Toast with Chat (With Header)
image

6.  Toast with Chat (Without Header)
image

7. Toast with Request (With Header)
image

8. Toast with Request (Without Header)
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: raaghu-elements/src/rds-toast/rds-toast.stories.tsx [hunk 1]
```diff
@@ -59,54 +60,124 @@ type Story = StoryObj<typeof RdsToast>;
...
+        iconFill: false,
+        borderColor: "primary",
+        layout : "request",
+        state: "basic"
     }
 } satisfies Story;
-toastWithoutHeader.parameters = { controls: { include: ['headerTitle', 'autohide', 'withIcon', 'delay', 'showHeader', 'message', 'colorVariant', 'iconName', 'iconColorvariant', 'iconHeight', 'iconWidth', 'iconFill'] } };
+toastWithRequest.parameters = { controls: { include: ["headerTitle", "message", "delay", "autohide", "withIcon", "showHeader", "iconName", "iconHeight", "iconWidth", "iconFill", "borderColor", "layout", "state"] } };
+
+// export const toastWithAutohide: Story = {
```
Comment (hemalatagaikwad) [inline raaghu-elements/src/rds-toast/rds-toast.stories.tsx]: remove commented code if not required in future

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
remove comments if not required

File: raaghu-elements/src/rds-toast/rds-toast.stories.tsx (hunk 1 above)
Comment (DeepashriDatar) [reply raaghu-elements/src/rds-toast/rds-toast.stories.tsx]: removed commented code which is not required.
in rds-toasts.tsx file there are 2-3 commented lines which are earlier used for color variant changes so kept that for reference.

**Comment by DeepashriDatar** [comment]
Code changes related to 'Color variant' should be done after Rahul pushed his changes of 'Color Variant' to development.

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
not working as expected
image

**Comment by DeepashriDatar** [comment]
Error resolved. Now its working

**Comment by DeepashriDatar** [comment]
Reopened

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
If colorVariant change filled color not change 
Is it possible?

**Comment by rahulp34** [comment]
Color Variants are added for icons also and icon fill control removed as per wireframe

image
image

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
@rahulp34  @DeepashriDatar please resolved build issue 
image

**Comment by rahulp34** [comment]
some args are not required, so removed it.
image

**Review by hemalatagaikwad — APPROVED** [review]
review and approve

**Review by hemalatagaikwad — APPROVED** [review]
review and approve

**Review by omkarchoudhari — APPROVED** [review]

File: raaghu-elements/src/rds-toast/rds-toast.stories.tsx (hunk 1 above)
Comment (omkarchoudhari) [reply raaghu-elements/src/rds-toast/rds-toast.stories.tsx]: @hemalatagaikwad @DeepashriDatar check this

Comment (hemalatagaikwad) [reply raaghu-elements/src/rds-toast/rds-toast.stories.tsx]: resolved

**Comment by rahulp34** [comment]
resolved

**Review by hemalatagaikwad — APPROVED** [review]
review and approve

**Comment by rahulp34** [comment]
resolve conflicts

**Pull request merged**